requirements/base.txt

```diff
@@ -1,2 +1,3 @@
 paho-mqtt
-prometheus-client
\ No newline at end of file
+prometheus-client
+re
```

Comment:
re is a builtin package, no need to install it from pypi

Reply:
Sorry @kpetremann – this was supposed to be a feature branch and the second commit was supposed to be only for me. Thanks for the help though! I'll update this PR.

Reply:
re should be placed before `import signal` for best practices purposes.
You can use [isort ](https://github.com/PyCQA/isort) to do it for you.